app/src/main/java/nerd/tuxmobil/fahrplan/congress/system/OnLocaleChangedReceiver.java

```diff
@@ -0,0 +1,34 @@
+package nerd.tuxmobil.fahrplan.congress.system;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import org.ligi.tracedroid.logging.Log;
+
+import nerd.tuxmobil.fahrplan.congress.notifications.NotificationHelper;
+
+public final class OnLocaleChangedReceiver extends BroadcastReceiver {
+
+    private static final String LOG_TAG = "onLocaleChanged";
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        if (intent == null) {
+            return;
+        }
+        String action = intent.getAction();
+        if (action == null) {
+            return;
+        }
+        if (!action.equals(Intent.ACTION_LOCALE_CHANGED)) {
```

Comment:
Avoid potential _NullPointerExceptions_ by changing this to `!Intent.ACTION_LOCALE_CHANGED.equals(action)`. This also allows removing the check above.

Reply:
Changed in 99f847a2f25ea475aab32219825d837468a00ca9.